api/src/main/java/org/apache/any23/extractor/Extractor.java

```diff
@@ -39,22 +38,6 @@
 
     /**
      * This interface specializes an {@link Extractor} able to handle
-     * {@link java.io.InputStream} as input format.
-     */
-    public interface ContentExtractor extends Extractor<InputStream> {
```

Comment:
@lewismc Why do you remove `ContentExtractor`? I assume that In case if content is neither html nor xml type that developer should create new extractor extending `Exctractor<Input>` directly. Am I right?